python/jiminy_py/src/jiminy_py/simulator.py

```diff
@@ -357,36 +397,50 @@ def reset(self, remove_all_forces: bool = False) -> None:
         self.engine.reset(False, remove_all_forces)
 
     def start(self,
-              q_init: np.ndarray,
-              v_init: np.ndarray,
-              a_init: Optional[np.ndarray] = None,
-              is_state_theoretical: bool = False) -> None:
+              q_init: Union[np.ndarray, Dict[str, np.ndarray]],
+              v_init: Union[np.ndarray, Dict[str, np.ndarray]],
+              a_init: Optional[Union[np.ndarray, Dict[str, np.ndarray]]] = None,
+              is_state_theoretical: Optional[bool] = None) -> None:
         """Initialize a simulation, starting from (q_init, v_init) at t=0.
 
-        :param q_init: Initial configuration.
-        :param v_init: Initial velocity.
-        :param a_init: Initial acceleration. It is only used by acceleration
-                       dependent sensors and controllers, such as IMU and force
-                       sensors.
-        :param is_state_theoretical: Whether the initial state is associated
+        :param q_init: Initial configuration (by robot if it is a dictionnary).
+        :param v_init: Initial velocity (by robot if it is a dictionnary).
+        :param a_init: Initial acceleration (by robot if it is a dictionnary).
+                       It is only used by acceleration dependent sensors and
+                       controllers, such as IMU and force sensors.
+        :param is_state_theoretical: In single robot simulations, whether the initial state is associated
                                      with the actual or theoretical model of
                                      the robot.
         """
         # Call base implementation
-        self.engine.start(q_init, v_init, a_init, is_state_theoretical)
```

Comment:
This method works for single-robot simulations, no need for anything fancy.

Reply:
I think that if you want to precise `is_state_theoretical` you need to send lists. If I remove the cases I have :
```
Engine.start(Engine, dict, dict, NoneType, bool)
did not match C++ signature:
    start(jiminy::Engine {lvalue} self, Eigen::Matrix<double, -1, 1, 0, -1, 1> q_init, Eigen::Matrix<double, -1, 1, 0, -1, 1> v_init, boost::python::api::object a_init=None, bool is_state_theoretical=False)
    start(jiminy::Engine {lvalue} self, boost::python::dict q_init_dict, boost::python::dict v_init_dict, boost::python::api::object a_init_dict=None)
```